src/e2e-test/java/io/cdap/plugin/pubsub/stepsdesign/PubSubSink.java

```diff
@@ -234,4 +237,33 @@ public void enterRuntimeArgumentValueForPubSubSinkPropertyTopicKey(String runtim
   public void clickOnPreviewDataForPubSubSink() {
     openSinkPluginPreviewData("GooglePublisher");
   }
+
+  @Then("Subscribe to the messages")
+  public void subscribeToTheMessages() throws InterruptedException {
+    TimeUnit time = TimeUnit.SECONDS;
+    time.sleep(60);
+    PubSubClient.subscribeAsyncExample(PluginPropertyUtils.pluginProp(ConstantsUtil.PROJECT_ID),
+                                       TestSetupHooks.pubSubSourceSubscription);
+  }
+
+
+  @And("Click on batch time and select format")
+  public void clickOnBatchTimeAndSelectFormat() {
+    Select select = new Select(PubSubLocators.batchTime);
+    select.selectByIndex(0);
+    Select selectformat = new Select(PubSubLocators.timeSelect);
+    selectformat.selectByIndex(1);
+    ElementHelper.clickOnElement(PubSubLocators.saveButton);
+  }
+
+  @And("Click on configure button")
+  public void clickOnConfigureButton() {
+    ElementHelper.clickOnElement(PubSubLocators.configButton);
+  }
+
+  @And("Click on pipeline config")
+  public void clickOnPipelineConfig() {
+    ElementHelper.clickOnElement(PubSubLocators.pipelineConfig);
```

Comment:
For these click steps can't we use the common steps from framework

Reply:
no we cannot use it from the common steps as it is not included in framework right now.